src/op_handlers/jump.rs

```diff
@@ -8,7 +8,7 @@ use crate::{opcode::Opcode, state::VMState};
 pub fn jump(vm: &mut VMState, opcode: &Opcode) -> Result<(), EraVmError> {
     let (src0, _) = address_operands_read(vm, opcode)?;
 
-    let next_pc = (src0.value & U256::from(u64::MAX)) - 1; // we subtract 1 because the pc will be incremented after this function
+    let next_pc = src0.value & U256::from(u64::MAX);
     vm.current_frame_mut()?.pc = next_pc.as_u64();
```

Comment:
These two could simply be:
```suggestion
    let next_pc = src0.value.low_u64();
    vm.current_frame_mut()?.pc = next_pc;
```